test/Sentry.Maui.Tests/SentryMauiLogcatsTests.cs

```diff
@@ -0,0 +1,66 @@
+using Microsoft.Extensions.Options;
+#if ANDROID
+using Sentry.Android;
+#endif
+
+namespace Sentry.Maui.Tests;
+
+public class SentryMauiLogcatsTests
+{
+    private class Fixture
+    {
+        public MauiAppBuilder Builder { get; }
+        public FakeTransport Transport { get; private set; } = new FakeTransport();
+        public InMemoryDiagnosticLogger Logger { get; private set; } = new InMemoryDiagnosticLogger();
+
+        public Fixture()
+        {
+            var builder = MauiApp.CreateBuilder();
+            builder.Services.AddSingleton(Substitute.For<IApplication>());
+
+            builder.Services.Configure<SentryMauiOptions>(options =>
+            {
+                options.Transport = Transport;
+                options.Dsn = ValidDsn;
+                options.AttachScreenshot = false; //Disable the screenshot attachment to have the logcat as primary attachment
+                options.Debug = true;
+                options.DiagnosticLogger = Logger;
+                options.AutoSessionTracking = false; //Get rid of session envelope for easier Assert
+                options.CacheDirectoryPath = null;   //Do not wrap our FakeTransport with a caching transport
+                options.FlushTimeout = TimeSpan.FromSeconds(10);
+            });
+            Builder = builder;
+        }
+    }
+
+    private readonly Fixture _fixture = new();
+
+#if __MOBILE__
+    [SkippableFact]
+    public void CaptureException_CheckLogcatType()
+    {
+#if __IOS__
+        Skip.If(true, "Doesn't support logcats");
+#endif
+
+        // Arrange
+        var builder = _fixture.Builder.UseSentry();
+        using var app = builder.Build();
+        SentryHint hint = null;
+        var options = app.Services.GetRequiredService<IOptions<SentryMauiOptions>>().Value;
+        var lcEvtProcessor = new LogCatAttachmentEventProcessor(options.DiagnosticLogger, options.Android.LogCatIntegration, options.Android.LogCatMaxLines)
+        var scope = new Scope(options);
+
+        // Act
+        lcEvtProcessor.Process(Arg.Any<SentryEvent>(), Arg.Do<SentryHint>(h => hint = h)).Returns(new SentryEvent());
```

Comment:
@jamescrosswell So here I'm testing the `LogCatAttachmentEventProcessor.Process()` method as you advised me, and making sure the log is attached as a `text/plain`. What do you think? Do you think it's enough, or is there something else you would test?